tad-lispy/npm-git-install
# Drop NodeGit

Use standard git binary instead.

This introduces an external dependency on `git` command. It's seems to be shortest path to fix long standing issues #5 and #7.

**How to test**

For now manual tests are required. This new code is not yet released to NPM so in order to test in your project, do:

``` sh
git clone https://github.com/lzrski/npm-git-install.git
cd npm-git-install
git checkout drop-node-git
npm link
```

then in a directory of your project with git dependencies:

``` sh
npm link npm-git-install
npm-git install
```

> **Important:** that the CLI command is now `npm-git install` (there is a space between `npm-git` and `install`, where a dash `-` used to be). You may have to update `scripts` section in  your `package.json`.

Change of a CLI command was done in preparation of new subcommand: `shrinkwrap`. See discussion here: lambdatastic/npm-git-install#3

Big thanks to @lambdatastic, @racerpeter and @jrolfs for setting me on the right path.
